src/coreclr/pal/tests/palsuite/runpaltests.sh

```diff
@@ -32,13 +32,6 @@ pushd $BUILD_ROOT_DIR
 
```

Comment:
For obtaining absolute path, we can make line 22-25 look similar to `src/coreclr/build-runtime.sh` and other scripts in the repo:

```sh
BUILD_ROOT_DIR="$(cd "$(dirname "$0")"; pwd -P)"
```
(a more elaborate variant of this is in eng/build.sh, which also accounts for symlink, but that's not supported by some other components of the build, so we usually don't care)